modules/ROOT/partials/configuration/sidebar_show.adoc

```diff
@@ -0,0 +1,27 @@
+[[sidebar_show]]
+== `+sidebar_show+`
+
+NOTE: This option is only available for TinyMCE 6.1 and later.
+
+This option allows you to show the specified sidebar on initialization.
```

Comment:
This also doesn't follow the style guide as we shouldn't be using pronouns: https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/CONTRIBUTING.md#pronouns

Reply:
```suggestion
This option allows the specified sidebar to be shown on editor initialization.
```